e2e/tests/content-type-builder/content-type.spec.js

```diff
@@ -16,6 +16,10 @@ const types = ['collection type', 'single type'];
 const type = types[0];
 
 test.describe(`Content Type Builder | Content-Type | ${type}`, () => {
+  test.beforeAll(async () => {
+    test.slow();
```

Comment:
"[test.slow()](https://playwright.dev/docs/api/class-test#test-slow-1) cannot be used in a beforeAll or afterAll hook. Use [test.setTimeout()](https://playwright.dev/docs/api/class-test#test-set-timeout) instead." – https://playwright.dev/docs/api/class-test#test-slow-1

Reply:
Yeah meant to put it outside forgot to make the change after reading the doc